wazo_sysconfd/plugins/systemd_services/services.py

```diff
@@ -66,12 +51,13 @@ def _run_action_for_service_validated(service, action):
         logger.debug("%s : return code %d", ' '.join(command), p.returncode)
 
         if p.returncode != 0:
-            raise HttpReqError(500, output)
+            raise exceptions.InternalServerErrorException(500, output)
     except OSError:
         logger.exception("Error while executing action")
-        raise HttpReqError(500, "can't manage services")
-
-    return output
+        raise exceptions.InternalServerErrorException(500, "can't manage services")
 
+    if isinstance(output, bytes):
+        return output.decode("utf-8")
+    else:
+        return output
```

Comment:
You should use `Popen`'s `encoding` argument instead, like you did in https://github.com/wazo-platform/wazo-sysconfd/pull/48

Reply:
If I do like you suggested, the unit tests `test_services.py` will fail with the error `No Call`

Reply:
Reverted

Reply:
This error was already fixed in the `fastapi` branch.